### PR TITLE
Individual case study

### PR DESCRIPTION
**Pull request by ashatat**
* change gatsbynode to accept templates as folders
* add style for InsightCard small
* add case1.md for gatsby to generate a template
* add individual case study template page

### CODE REVIEW

File: src/templates/case-study/style.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles(theme => ({
+  container: {
+    padding: theme.spacing(5, 0, 10, 0),
```
Comment (JoseChungFOS) [inline src/templates/case-study/style.js]: The container is a bit too wide compared to the design - I played around with it a bit and looks like this is about right - see what you think!
```suggestion
    padding: theme.spacing(5, 5, 10, 5),
```

File: src/templates/case-study/style.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    },
+    '& > h6:first-child': {
+      fontSize: '1.3rem',
+      [theme.breakpoints.down('xs')]: {
+        fontSize: '1.1rem',
+        color: theme.palette.common.black,
+      },
+    },
+  },
+  image: {
```
Comment (JoseChungFOS) [inline src/templates/case-study/style.js]: The image has a border radius coming from the previewCompatibleImage component. We should probably take it out of the image component, but for now can you just add `borderRadius: 0` to the image style

File: src/templates/case-study/style.js (hunk 1 above)
Comment (ashatat) [reply src/templates/case-study/style.js]: done

File: src/templates/case-study/style.js (hunk 2 above)
Comment (ashatat) [reply src/templates/case-study/style.js]: done

**Pull request merged**